app/views/items/_form.html.erb

```diff
@@ -0,0 +1,9 @@
+<%= form_for [user, user.items.new], remote: true do |f| %>
```

Comment:
It did work for me, but I'll admit it looks ugly to me now.  This would probably be better served as an instance variable from the controller, right? Especially since now the form partial can't be used for editing items if I ever add that to the app later.